repos/spack_repo/builtin/packages/py_distributed/package.py

```diff
@@ -44,6 +45,8 @@ class PyDistributed(PythonPackage):
     depends_on("python@3.8:", when="@2022.2.1:", type=("build", "run"))
     # https://github.com/dask/distributed/pull/6605
     depends_on("python@:3.11", when="@:2022.6.0", type=("build", "run"))
+    depends_on("python@3.10:", when="@2025.7.0:", type=("build", "run"))
+
     depends_on("py-setuptools", type="build")
     depends_on("py-setuptools@62.6:", type="build", when="@2023.4.1:")
 
```

Comment:
version dep should be updated, see if we can't use exactly 0.28 and 0.29. May be able to add the build-tools tag to let Spack use multiple different versions in the same DAG

Reply:
TIL `build-tools`. That seems like the perfect fix and what I should have done previously had I known. Ty